python/ray/data/dataset_pipeline.py

```diff
@@ -1123,8 +1123,9 @@ def from_iterable(
         return DatasetPipeline(iterable, False, length=length)
 
     def __repr__(self) -> str:
-        return "DatasetPipeline(num_windows={}, num_stages={})".format(
-            self._length, 1 + len(self._stages)
+        return (
+            f"DatasetPipeline(num_windows={self._length}, "
+            f"num_stages={len(self._stages)})"
```

Comment:
dumb question - why we have `1 +` before? is it for the read stage?

Reply:
I assume we might have treated "window" or "repeat" as a stage.